schemas/iso19139/src/main/plugin/iso19139/schematron/schematron-rules-url-check.sch

```diff
@@ -13,7 +13,7 @@
     <sch:pattern>
         <sch:title>$loc/strings/invalidURLCheck</sch:title>
         <!-- Check specification names and status -->
-        <sch:rule context="//gmd:linkage//gmd:URL">
+        <sch:rule context="//gmd:linkage//gmd:URL[starts-with(text(), 'http')]">
```

Comment:
Should be better to lower case the text?

Reply:
"URLs in general are case-sensitive (with the exception of machine names). There may be URLs, or parts of URLs, where case doesn't matter, but identifying these may not be easy. Users should always consider that URLs are case-sensitive. " Cf. https://www.w3.org/TR/WD-html40-970708/htmlweb.html